src/stitches.config.ts

```diff
@@ -41,6 +42,7 @@ const spaces = {
   xs: '4px',
   sm: '8px',
   md: '16px',
+  md2: '18px',
```

Comment:
Not sure how best to deal with these custom sizing properties. We might need to agree on what the best practice should be but I do recognize that you need to be able to specify numbers.